versions/v6/templates/story/related-stories.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$related_stories = related_stories_get_stories( $post->ID );
+
+if ( $related_stories ) :
+?>
+<aside>
+	<h2>Related Stories</h2>
+	<?php foreach( $related_stories as $story ) : ?>
+		<article class="d-block mb-4">
+			<img alt="" class="img-fluid" src="<?php echo $story['thumbnail']; ?>">
+			<a class="h6 text-secondary stretched-link" href="<?php echo $story['url']; ?>"><?php echo $story['title']; ?></a>
```

Comment:
Either add `.position-relative` to each `<article>` or remove `.stretched-link` + wrap each link around the story thumbnails; the stretched links are bleeding out to the wrapper `.col`